### ISSUE TITLE
quit+delete group but group remains in chat window

### ISSUE THREAD

**Issue by CaptainCoward** [zhukov/webogram#1187]
any solution for that issue? I also tried deleting history but the group just stays in my "chat window".

**Comment by zhukov** [comment]
Can you record a video? I can't reproduce it.
Also please open javascript error console after you peform delete and check if you see any errors.

**Comment by SKUndef** [comment]
I can confirm, I have a group deleted (and not shown in Android app), but still present in chat list. If i try to send a message in the group, it throws an error.

![schermata da 2016-07-18 16-16-37](https://cloud.githubusercontent.com/assets/5783302/16917898/c21ddd00-4d03-11e6-9583-80eb6c652ced.png)

**Comment by zhukov** [comment]
what are the steps to reproduce?

**Comment by fref** [comment]
I have the same issue for two groups. 
I don't know what the steps to reproduce are, but it's annoying. 
The groups are not shown/were properly removed from the Android app view, but not from the web-version.
Both groups were created by somebody else. I left them after they were no longer relevant but the web-version view remains.
There is no option to leave the groups anymore, the console does not show any error.
![2017-01-16 14_13_45-telegram web](https://cloud.githubusercontent.com/assets/130824/21984434/247b2f94-dbf6-11e6-9b86-934d32d11e85.png)

**Comment by fref** [comment]
When trying to send a message to the group
![2017-01-16 14_17_21-telegram web](https://cloud.githubusercontent.com/assets/130824/21984528/9cb7c724-dbf6-11e6-8d41-efe08de987c1.png)

**Comment by stek29** [comment]
Does it show after relogin?

**Comment by fref** [comment]
Yes, I did a full logout, I cleared the session storage, app storage, ... re-logged in, it still showed the two offending groups.

**Comment by stek29** [comment]
Do you have those in TDeskop?

**Comment by fref** [comment]
The groups appeared in the Portable version of TDesktop, but I could
"Delete and exit" them from there and when I fired up the Web version
afterwards, they seem to have disappeared.
